conf/application.conf

```diff
@@ -70,7 +70,7 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.graphite.GraphiteMetricsModu
 # Provides an implementation and configures all filters required by a Platform frontend microservice.
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.FrontendModule"
 
-play.filters.headers.contentSecurityPolicy = "default-src 'self' 'unsafe-inline' localhost:9000 localhost:9032 www.google-analytics.com data: ; script-src 'self' 'unsafe-inline' 'unsafe-eval' localhost:9032 localhost:9250  https://tagmanager.google.com/ https://www.googletagmanager.com/ www.google-analytics.com; style-src 'self' 'unsafe-inline' localhost:9032 https://tagmanager.google.com/; img-src 'self' data: https://ssl.gstatic.com/ www.google-analytics.com; font-src 'self' data: https://fonts.googleapis.com/; frame-src 'self' https://www.googletagmanager.com;"
+play.filters.headers.contentSecurityPolicy = "default-src 'self' 'unsafe-inline' localhost:9000 localhost:9032 www.google-analytics.com data: ; script-src 'self' 'unsafe-inline' 'unsafe-eval' localhost:9032 localhost:9250  https://tagmanager.google.com/ https://www.googletagmanager.com/ www.google-analytics.com; style-src 'self' 'unsafe-inline' localhost:9032 https://tagmanager.google.com/; img-src 'self' data: https://ssl.gstatic.com/ www.google-analytics.com; font-src 'self' data: https://fonts.googleapis.com/; frame-src 'self' https://www.googletagmanager.com; *.optimizely.com optimizely.s3.amazonaws.com"
```

Comment:
Missing https://cdn.optimizely.com/js in `script-src`
this is wrong `*.optimizely.com optimizely.s3.amazonaws.com` It's not specified for which part of CSP is used.
This tool can help you with CSP header: https://report-uri.com/home/generate